packages/components/organisms/f-content-cards/src/components/cardTemplates/StampCard1.vue

```diff
@@ -154,6 +223,40 @@ export default {
 
         onClickContentCard () {
             this.emitCardClick(this.card);
+        },
+
+        /**
+         * Takes the locale and lazyloads the correct date locale for the date-fns library
+         * @param locale
+         * @returns {
+         *  Promise<module:date-fns/locale/da> |
+         *  Promise<module:date-fns/locale/en-AU> |
+         *  Promise<module:date-fns/locale/en-GB> |
+         *  Promise<module:date-fns/locale/en-NZ> |
+         *  Promise<module:date-fns/locale/es> |
+         *  Promise<module:date-fns/locale/it> |
+         *  Promise<module:date-fns/locale/nb>
+         * }
+         */
+        getDateFnsLocale (locale) {
+            switch (locale) {
+                case 'da-DK':
+                    return import(/* webpackChunkName: "date-fns-locale-da" */ 'date-fns/locale/da');
```

Comment:
Nice :)

Reply:
yes thanks to @DamianMullins for the naming suggestion.